sdk/storage/azure-storage-blob/HISTORY.md

```diff
@@ -6,6 +6,8 @@
 
 - `set_container_access_policy` has required parameter `signed_identifiers`.
 - NoRetry policy has been removed. Use keyword argument `retry_total=0` for no retries. 
+- Now only the async clients `BlobServiceClient`, `ContainerClient`, `BlobClient`, and `LeaseClient` can be imported 
```

Comment:
Change phrasing to something like:

"Removed types that were accidentally exposed from two modules. Only BlobServiceClient, ContainerClient, BlocBlient and LeaseClient should be imported from azure.storage.blob.aio".

?